Core/HLE/HLE.cpp

```diff
@@ -83,12 +83,12 @@ struct HLEMipsCallStack {
 			u32_le func;
 			u32_le actionIndex;
 			u32_le argc;
-		};
+		} vars;
 		struct {
 			u32_le ra;
 			u32_le v0;
 			u32_le v1;
-		};
+		} regs;
 	};
 };
 
```

Comment:
is this struct ever read by non native code ? because, as the name suggests, it is HLE, so even if the data lives in psp memory, it might not necessarily need to be little endian.
I did test it briefly with native types and nothing collapsed, so maybe that's the much simpler solution to this as opposed to naming the structs ?

Reply:
Hm, I think that might be fine indeed...

Reply:
oh yeah savestates. even if they might not be entirely endian safe right now, let's not make the matter worse.
ok better to leave it like this then I guess...